### ISSUE TITLE
Allow linking to K markdown sources in tutorial webpages

### ISSUE THREAD

**Issue by Baltoli** [runtimeverification/k#3280]
A community user reports that [Lesson 1.8](https://kframework.org/k-distribution/k-tutorial/1_basic/08_literate_programming/) links to the webpage for Lesson 1.3 (in exercise 2), rather than the Markdown source as would be appropriate.

Instead of linking to `https://kframework.org/k-distribution/k-tutorial/1_basic/03_parsing/` in exercise 2, we should link to `https://github.com/runtimeverification/k/blob/master/k-distribution/k-tutorial/1_basic/03_parsing/README.md`

**Comment by yiyi-wang-rv** [comment]
Hi @Baltoli , one solution that I can think of is we change this line:

```
2. Compile [Lesson 1.3](../03_parsing/README.md) as a K definition. Identify
why it fails to compile. Then pass an appropriate `--md-selector` to the
compiler in order to make it compile.
```

to

```
2. Compile [Lesson 1.3](https://raw.githubusercontent.com/runtimeverification/k/master/k-distribution/k-tutorial/1_basic/03_parsing/README.md) as a K definition. Identify
why it fails to compile. Then pass an appropriate `--md-selector` to the
compiler in order to make it compile.
```

so we use the absolute URL https://raw.githubusercontent.com/runtimeverification/k/master/k-distribution/k-tutorial/1_basic/03_parsing/README.md instead of `../03_parsing/README.md`

Thanks

**Comment by Baltoli** [comment]
That sounds good!

**Comment by Robertorosmaninho** [comment]
Solved in #3292
